cmd/tnf/generate/config/config.go

```diff
@@ -86,7 +86,7 @@ func createConfiguration() {
 	createMenu := []configOption{
 		{Option: cnfResources, Help: cnfResourcesHelp},
 		{Option: exceptions, Help: exceptionsdHelp},
-		{Option: collector, Help: collectordHelp},
+		// {Option: collector, Help: collectordHelp},
```

Comment:
What's the purpose of retaining commented lines? Git won't forget :-)

Reply:
They will be uncommented soon when we enable the collector.  I'm fine with keeping them here.